### PR TITLE
chore(backstage): update to backstage 1.18.4

### LINKED ISSUE #593: Not fully upgraded to Backstage 1.18
### Describe the bug

Based on the latest upgrade to Backstage 1.18 version, the merged [PR](https://github.com/janus-idp/backstage-showcase/pull/577) did not contain all the [required changes](https://backstage.github.io/upgrade-helper/?from=1.17.5&to=1.18.4).

* Missing a new command `fix` in the `package.json` file.
* Missing the `databaseManager` change in the `backend/src/index.ts` file.

### PR DESCRIPTION
**Pull request by Zaperex**
## Description

Finish upgrade to 1.18.x (upgrades to 1.18.4)

## Which issue(s) does this PR fix

- Fixes #593 

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

**Comment by schultzp2020** [comment]
Did you include the non-core backstage plugins in the update command? https://github.com/janus-idp/backstage-showcase/blob/main/CONTRIBUTING.md#updating-backstage-dependencies

**Comment by Zaperex** [comment]
Ah I did not, let me run the command and see if anything changed

File: .changeset/rare-laws-deny.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+'app': major
+'backend': minor
+---
+
+Update backstage to [1.18.4](https://github.com/backstage/backstage/releases/tag/v1.18.4)
+
```
Comment (schultzp2020) [inline .changeset/rare-laws-deny.md:7]:
```suggestion

**BREAKING**

```

**Review by schultzp2020 — CHANGES_REQUESTED** [review]

**Review by schultzp2020 — APPROVED** [review]
LGTM

File: .changeset/rare-laws-deny.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+**BREAKING**
+
+Updated Tekton plugin now expects a `janus-idp.io/tekton` annotation instead of a `janus-idp.io/tekton-enabled` annotation on the catalog entities.
```
Comment (christophe-f) [inline .changeset/rare-laws-deny.md:10]: This should be part of another PR as not related to Backstage 1.18.x

**Review by christophe-f — APPROVED** [review]

File: .changeset/rare-laws-deny.md (hunk 2 above)
Comment (Zaperex) [reply .changeset/rare-laws-deny.md:10]: So do you want to revert the upgrades to the `@janus-idp` packages and only upgrade the `@backstage` packages?

Comment (Zaperex) [reply .changeset/rare-laws-deny.md:10]: Reverted the commit https://github.com/Zaperex/backstage-showcase/commit/495bd3d3d1e061420e6d15ab24c20217f8fd311b
